### PR TITLE
Fix trim multiqc

### PR DESCRIPTION
**Pull request by rannick**
Update modules, and make sure multiqc displays the QC results properly 

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/rnafusion/tree/master/.github/CONTRIBUTING.md)
- [x] If necessary, also make a PR on the nf-core/rnafusion _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: assets/multiqc_config.yml [hunk 1]
```diff
@@ -4,9 +4,39 @@ report_comment: >
...
+  - fastqc:
+      name: "FastQC (trimmed)"
+      info: "This section of the report shows FastQC results after adapter trimming."
+      path_filters:
+        - "*_trimmed*.zip"
+  - arriba:
+      name: "Arriba visualisation"
+      info: "This section of the report shows Arriba visualisation after FusionInspector analysis."
+      path_filters:
+        - "*.pdf"
```
Comment (fevac) [inline assets/multiqc_config.yml:42]:
```suggestion
```
not in use. remove

File: assets/multiqc_config.yml [hunk 2]
```diff
@@ -4,9 +4,39 @@ report_comment: >
...
+  - picard
+  - arriba
+
+module_order:
+  - fastqc:
+      name: "FastQC (raw)"
+      info: "This section of the report shows FastQC results before adapter trimming."
+      path_filters:
+        - "*.zip"
+  - fastp
```
Comment (fevac) [inline assets/multiqc_config.yml:32]:
```suggestion
  - fastp
  - fastqc:
      name: "FastQC (raw)"
      info: "This section of the report shows FastQC results before adapter trimming."
      path_filters:
        - "*.zip"
```
we could move this up just before fastqc so we have both fastqc results one after another. it would seem clearer to me, but up to you!

File: conf/modules.config [hunk 3]
```diff
@@ -68,12 +68,18 @@ process {
...
         ext.when         = { !params.skip_qc }
+        publishDir = [
+            path: { "${params.outdir}/fastqc" },
+            mode: params.publish_dir_mode,
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+        ]
     }
 
-    withName: FASTQC_FOR_TRIM {
+    withName: FASTQC_FOR_FASTP {
```
Comment (fevac) [inline conf/modules.config:78]: why the name change?

File: modules/nf-core/samtools/index/tests/csi.nextflow.config [hunk 4]
```diff
@@ -0,0 +1,7 @@
+process {
+
+    withName: SAMTOOLS_INDEX {
+        ext.args = '-c'
```
Comment (fevac) [inline modules/nf-core/samtools/index/tests/csi.nextflow.config:4]: why this change here?  or is it part of modules update?

**Review by fevac — CHANGES_REQUESTED** [review]
don't forget to update the changelog. It would be nice to add the summary of all changes. To me this pr is a bit of a mix, for what I see:
- changes multiqc report
- updates versions for multiqc and samtools?
- updates in trim workflow

File: conf/modules.config (hunk 3 above)
Comment (rannick) [reply conf/modules.config:78]: FASTQC_FOR_TRIM does not exist, it was related to a non existent module. The right modules is FASTQC_FOR_FASTP is the correct name

File: modules/nf-core/samtools/index/tests/csi.nextflow.config (hunk 4 above)
Comment (rannick) [reply modules/nf-core/samtools/index/tests/csi.nextflow.config:4]: it is part of the modules update, it is a test

**Review by fevac — APPROVED** [review]
🚀
